help/destinations/catalog/advertising/acxiom-audience-distribution.md

```diff
@@ -0,0 +1,123 @@
+---

```

Comment:
I just noticed the assets folder is still using the wrong path. Your destination page is in /catalog/advertising, but your assets are in /catalog/data-partner. Please move the assets folder to the advertising category and update all your image links to the correct path.

Reply:
Files have been moved to the new location, and links have been updated.